akka-remote/src/test/scala/akka/remote/artery/RemoteDeathWatchSpec.scala

```diff
@@ -56,7 +56,7 @@ class RemoteDeathWatchSpec
 
   override def expectedTestDuration: FiniteDuration = 120.seconds
 
-  "receive Terminated when system of de-serialized ActorRef is not running" in {
+  "receive Terminated when system of de-serialized ActorRef is not running" taggedAs GHExcludeTest in {
```

Comment:
also seen (rarely) on jenkins so perhaps not necessary to exclude on GHA (https://github.com/akka/akka/issues/30454)

Reply:
removed